src/main/scala/cromwell/binding/Call.scala

```diff
@@ -34,6 +35,27 @@ case class Call(alias: Option[String], task: Task, inputMappings: Map[String, Wd
    * @return Map[String, WdlType] representing each task-local input
    */
   def unsatisfiedInputs: Map[String, WdlType] = task.inputs.filterNot { case (k, v) => inputMappings.contains(k) }
-  
+
   override def toString: String = s"[Call name=$name, task=$task]"
+
+  /** Find all calls upon which this call immediately depends, i.e. the result of this
+    * does not include transitive dependencies.  Currently this only works for member
+    * access expressions with a literal LHS, e.g.:
+    * 
+    * {{{
+    *   call cgrep {
+    *     input: in_file=ps.procs
+    *   }
+    * }}}
+    * 
+    */
+  def prerequisiteCalls(): Traversable[Call] = {
```

Comment:
I think the two problems with this algorithm in the short-to-mid term (not an issue now) is that we need to check more than just MemberAccess's, but MemberAccess's encompass a vast majority of ways we model dependencies, so this is fine for at least the next few sprints.

The real brittle part I think is the `parent.get.asInstanceOf[Workflow].calls`.  At some point soon we need to really consider how to get all calls in scope of this call, and in general talk about what it means to be in scope.